frontend/src/Frontend/Page/ReqKey.hs

```diff
@@ -127,6 +128,7 @@ requestKeyResultPage netId cid (CommandResult rk txid pr g logs pcont meta evs)
     el "h2" $ text "Transaction Results"
     elAttr "table" ("class" =: "ui definition table") $ do
       el "tbody" $ do
+        tfield "Detail" $ txDetailLink netId (requestKeyToB16Text rk) (requestKeyToB16Text rk)
```

Comment:
Instead of adding this field can we change the existing request key field into a link.  That would keep the page more concise while adding the new functionality.